packages/search/src/features/registration/birth/service.ts

```diff
@@ -128,25 +89,38 @@ async function updateEvent(bundle: fhir.Bundle, authHeader: string) {
   body.type = getTypeFromTask(task)
   body.modifiedAt = Date.now().toString()
 
+  if (body.type === REGISTERED_STATUS) {
+    const childSection = findCompositionSection('child-details', composition)
+    const patientId = urlReferenceToUUID(childSection.entry[0].reference)
+    const patient = findPatientEntryById(bundle, patientId)
+
+    body.childIdentifier =
+      patient.identifier && getSubmittedIdentifier(patient.identifier)
+    body.registrationNumber =
+      registrationNumberIdentifier && registrationNumberIdentifier.value
+  }
```

Comment:
if we are doing this only for a "REGISTERED" declaration then won't the registrationNumber not get saved for "CERTIFIED"/"ISSUED" ones? I'm guessing at the end of this handler, we replace the previous elastic document with the newly created one

Reply:
My bad. I just thought this would only be saved for `REGISTERED` status like `REJECTED`.